graylog2-web-interface/src/views/components/aggregationwizard/grouping/GroupingConfiguration.tsx

```diff
@@ -32,6 +33,45 @@ type Props = {
   index: number,
 }
 
+type SkipEmptyValuesHoverForHelpProps = {
+  children: React.ReactNode,
+  title: string,
+};
+
+const SkipEmptyValuesHoverForHelp = styled((props: SkipEmptyValuesHoverForHelpProps) => <HoverForHelp {...props} />)`
+  margin-left: 5px;
+`;
+
+const SkipEmptyValuesCheckbox = styled(Checkbox)`
+  &.checkbox {
+    padding-top: 0;
+  }
+`;
+const SkipEmptyValuesLabel = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+type SkipEmptyValuesPropes = {
+  index: number,
+}
+const SkipEmptyValues = ({ index }: SkipEmptyValuesPropes) => (
+  <Field name={`groupBy.groupings.${index}.skipEmptyValues`}>
+    {({ field: { name, value, onChange } }) => (
+      <SkipEmptyValuesCheckbox onChange={() => onChange({ target: { name, value: !value } })} checked={value ?? false}>
+        <SkipEmptyValuesLabel>
+          Skip Empty Values
+          <SkipEmptyValuesHoverForHelp title="Skip Empty Values">
+            When this is enabled, messages which do not contain the configured fields will be skipped.
+            <p />
+            Otherwise an &quot;(Empty Value)&quot; bucket will be created.
+          </SkipEmptyValuesHoverForHelp>
+        </SkipEmptyValuesLabel>
+      </SkipEmptyValuesCheckbox>
+    )}
```

Comment:
Have you considered unifying the layout with e.g. the visualization config checkboxes?

![image](https://github.com/Graylog2/graylog2-server/assets/46300478/5346c207-5eb1-4c1f-9a98-2bd53a009e6e)